hydra-node/src/Hydra/HeadLogic/Outcome.hs

```diff
@@ -143,6 +143,7 @@ data StateChanged tx
       , participants :: [OnChainId]
       }
   | TxInvalid {headId :: HeadId, utxo :: UTxOType tx, transaction :: tx, validationError :: ValidationError}
+  | ClearLocalState {headId :: HeadId, snapshotNumber :: SnapshotNumber}
```

Comment:
I like the minimalism of this `StateChanged` value and that it contains the `SnapshotNumber` to reference a `SnapshotConfirmed` event, but is this the best name for it? Sounds a lot like a "command" and how does a snapshot number relate to "clear local state"?

Could: find a better name

Reply:
how about: `LocalStateResetToSnapshot` ?

Reply:
going with `LocalStateCleared`